convert.py

```diff
@@ -1123,7 +1124,7 @@ def main(args_in: list[str] | None = None) -> None:
     parser.add_argument("--outtype",     choices=output_choices, help="output format - note: q8_0 may be very slow (default: f16 or f32 based on input)")
     parser.add_argument("--vocab-dir",   type=Path,              help="directory containing tokenizer.model, if separate from model file")
     parser.add_argument("--outfile",     type=Path,              help="path to write to; default: based on input")
-    parser.add_argument("model",         type=Path,              help="directory containing model file, or model file itself (*.pth, *.pt, *.bin)")
+    parser.add_argument("model",         type=Path,              help="directory containing model file, or model file itself (*.pth, *.pt, *.bin, *.safetensors)")
```

Comment:
Possibly fix the weird wording here as well since you're in the neighborhood. I'd say something like "*directory containing model file or files (...*"

Reply:
The wording here makes sense to me. The argument does not have to be (and in some cases cannot be) a directory, it can be a path to the model file itself (which the globs try to guess otherwise).

Reply:
Fair enough, disregard what I said.